opentelemetry-semantic-conventions/scripts/templates/header_resource.rs

```diff
@@ -7,14 +7,31 @@
 //!
 //! ## Usage
 //!
+//! [`tracing`]:
+//! 
 //! ```
+//! use opentelemetry_semantic_conventions as semconv;
+//! use tracing::span;
+//!
+//! let span = span!(
+//!     LEVEL::INFO,
+//!     "handle_request",
+//!     { semconv::resource::SERVICE_NAME = "my-service" },

```

Comment:
I am trying to understand this line here.. Is this adding service name and ns to every span as attributes? If yes, that is not something we should show/encourage, as Resource is expected to be attached to the Provider level *not* to each span/log/metric. (In somebackends, it maybe ultimately stored for each span/log, but thats at exporter level//backend level)

Reply:
OK, just removed.